task_02/nikolai_chernyshov/main.cpp

```diff
@@ -0,0 +1,175 @@
+#include <memory>
+#include <cstdlib>
+#include <iostream>
+#include <fstream>
+
+struct edge {
```

Comment:
`Struct` и `Class` обычно называются с большой буквы
```suggestion
struct Edge {
```